codebuild/spec/buildspec_ubuntu_integ_awslc.yml

```diff
@@ -0,0 +1,56 @@
+---
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+# this file except in compliance with the License. A copy of the License is
+# located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and
+# limitations under the License.
+version: 0.2
+
+env:
+  variables:
+    # CODEBUILD_ is a reserved namespace.
+    CB_BIN_DIR: "./codebuild/bin"
+
+phases:
+  install:
+    runtime-versions:
+      python: 3.7
+    commands:
+      - echo Entered the install phase...
+      - add-apt-repository ppa:ubuntu-toolchain-r/test -y
+      # Add repo to get latest golang version. https://github.com/golang/go/wiki/Ubuntu
+      - add-apt-repository ppa:longsleep/golang-backports
+      - apt-get update -o Acquire::CompressionTypes::Order::=gz
+      - apt-get update -y
+      - apt-get install -y --no-install-recommends gcc g++ gcc-4.8 g++-4.8 gcc-6 g++-6 gcc-9 g++-9
+      # Don't install old clang and llvm if LATEST_CLANG is enabled, handle it in install_clang.sh instead
+      - |
+        if expr "${LATEST_CLANG}" != "true" >/dev/null; then
+          apt-get install -y --no-install-recommends clang-3.9 llvm-3.9;
+        fi
+      - apt-get install -y --no-install-recommends indent kwstyle lcov libssl-dev m4 make net-tools nettle-bin nettle-dev pkg-config psmisc python3-pip shellcheck sudo tcpdump unzip valgrind zlib1g-dev zlibc cmake tox libtool ninja-build quilt golang-go
```

Comment:
We should make sure we meet all requirements specified here: https://github.com/awslabs/aws-lc/blob/main/third_party/boringssl/BUILDING.md#build-prerequisites
Ubuntu might come pre-installed with these but who knows. I would like to see at least:
* Make sure `Perl` is installed
* Make sure CMake version >= 3.0

`libunwind` is not required here.

Reply:
* `Perl` is preinstalled in `ubuntu 18.04`.
* `cmake_minimum_required` in aws-lc guarantees fail earlier.